FileConventions.Test/FileConventions.Test.fsproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+    <GenerateProgramFile>false</GenerateProgramFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="UnitTest1.fs" />

```

Comment:
let's rename this file to not have the default name

Reply:
Is this file going to include all the tests, like what we did in commitlintplugins.test or just the HasShebang function's tests?

Reply:
there's no right or wrong answer to that question; it probably can have all tests for now, until it becomes very big, in which case it would be better to split; it's a shades of grey answer vs black&white

Reply:
Right, Done